app-text/nfoview/nfoview-2.1.ebuild

```
@@ -0,0 +1,78 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit desktop python-single-r1 xdg virtualx
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/otsaloma/nfoview.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/otsaloma/nfoview/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Simple viewer for NFO files, which are ASCII art in the CP437 codepage"
+HOMEPAGE="https://otsaloma.io/nfoview/"
+
+LICENSE="GPL-3+"
+SLOT="0"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+BDEPEND="
+	${PYTHON_DEPS}
+	sys-devel/gettext
+	$(python_gen_cond_dep '
+		dev-python/pytest[${PYTHON_USEDEP}]
+	')
+"
+DEPEND="$(python_gen_cond_dep 'dev-python/pygobject:3[${PYTHON_USEDEP}]')"
+RDEPEND="${DEPEND}
+	${PYTHON_DEPS}
+	media-fonts/cascadia-code
+	gui-libs/gtk:4[introspection]"
+
+src_compile() {
+	true
+}
+
+src_install() {
+	sed -i 's,return Path(__file__).parent.parent / "data",return Path(__file__).parent,1' nfoview/paths.py
+	sed -i 's,return Path(__file__).parent.parent / "locale",return Path("/usr/share/locale/"),1' nfoview/paths.py
```

```suggestion
	sed -e 's,return Path(__file__).parent.parent / "data",return Path(__file__).parent,1' \
		-e 's,return Path(__file__).parent.parent / "locale",return Path("/usr/share/locale/"),1' \
		-i nfoview/paths.py || die
```

Though at this point I'd consider a patch.

---

fixed